integration-tests/broken-app/index.js

```diff
@@ -0,0 +1,37 @@
+const { registerHtml, start } = require('../../src/tram-one')
+
+const html = registerHtml({
+	title: require('./title')
+})
+
+/**
+ * main app to power integration tests
+ */
+const app = () => {
+	return html`
+		<main>
+			<title />
+		</main>
+	`
+}
+
+const startApp = container => {
+	if (container === undefined) {
+		container = document.createElement('div')
+		container.id = 'app'
+	}
+
+	start(app, container)
+
+	return {
+		container
+	}
+}
+
+if (document.querySelector('#parcel-page')) {
+	startApp('#parcel-page')
+}
+
+module.exports = {
+	app, startApp
+}
```

Comment:
This is copied-pasted a lot, I should make this a helper function or something (if exports can be put in helper functions? maybe just the `startApp` :thinking: )